### ISSUE TITLE
Errors in Post Install Hook Are Not Logged

### ISSUE THREAD

**Issue by robscott** [reactiveops/reckoner#59]
It seems like whatever output comes from a post install hook doesn't actually make it to stdout. That means if you happening to be referencing an invalid path, there will be nothing in the Reckoner output to indicate there's a problem with your config.

**Comment by endzyme** [comment]
Quick clarification - would you expect it to always return stderr or only when exitcode != 0

**Comment by endzyme** [comment]
Other questions on expectations

Today all hooks just blindly accept that they can fail but continue with the chart installation. 

1. should pre-install hooks stop a chart from running
2. should post-install hooks stop the next chart from running
3. if there are three hooks in pre-install, and hook 2 fails, should hook 3 run?

TL;DR; what should reasonable error handling look like for hooks?
